code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -39,6 +39,14 @@
 		r_store,
 		w_uniform)
 
+/mob/living/carbon/human/get_nonclothing_items() //pockets, hands, belt

```

Comment:
So then this should just be get_all_slots() - get_clothing_items().
It's also not used anywhere.